taskvine/src/tools/vine_plot_txn_log

```diff
@@ -435,6 +435,12 @@ class TxnPlot:
                 "outputs to manager": 'C5',
                 }
 
+        # dask legend
+        self.legend_dask = {
```

Comment:
I suggest:
Rename self.legend to self.legend_cb

```python

self.legend_dask = dict(self.legend_cb)
self.legend_dask.update(
{
 "task executing": '#7fc .... etc.
})
self.legend = self.legend_cb
```

Then, if `--dask-colors` is specified, simply assign `self.legend = self.legend_dask`. With that I don't think you need to change any of the functions `bh`,  `set_legend`, or  `plot`, nor create `plot_dask`.

Reply:
That matplotlib palette is supposed to be friendly to **c**olor**b**lind persons. (e.g. https://viscid-hub.github.io/Viscid-docs/docs/dev/styles/tableau-colorblind10.html)

Reply:
Great! Thanks!

Reply:
This may have conflicts with my current idea... I changed `bh` because the edgecolor needs to align with Dask's, and I changed `plot_dask`, because in Dask's figure there are only two kinds of bars and we don't need to plot the worker lifetime, and `set_legend` for the same reason.

![image](https://github.com/cooperative-computing-lab/cctools/assets/142265839/d2662af8-7ec8-429c-9796-bfd354b0ac1f)